packages/e2e-tests/src/Yarn.ts

```diff
@@ -65,18 +69,45 @@ class Yarn {
 		return this;
 	}
 
+	private saveArtifact(
+		file: string,
+		report: {
+			name: string;
+			expiry: number; // in days
+		}
+	): boolean | Promise<boolean> {
+		const tarball = this.tar.create(file);
+
+		if (!tarball) {
+			return false;
+		}
+
+		const gpg = this.gpg.encrypt(tarball);
+
+		if (!gpg) {
+			return false;
+		}
+
+		const fileName = path.basename(gpg);
+		const rootDir = path.dirname(gpg);
+
+		return this.artifact.save(fileName, rootDir, report.name, report.expiry);
```

Comment:
previously, in `saveReport()` & `saveTestResults()`, `this.artifact.save()` was called using `await` but now it is not?

Reply:
So the chain of "reaching" this line is as follows:

`Action.ts`:

https://github.com/eventespresso/actions/blob/751d4a85eac61054f446442926dc0354c55bef14/packages/e2e-tests/src/Action.ts#L44

from there, we call `saveReport()` and `saveTestResults()`

https://github.com/eventespresso/actions/blob/751d4a85eac61054f446442926dc0354c55bef14/packages/e2e-tests/src/Yarn.ts#L64-L67

those respective functions always invoke `saveArtifact()` last

https://github.com/eventespresso/actions/blob/751d4a85eac61054f446442926dc0354c55bef14/packages/e2e-tests/src/Yarn.ts#L97-L111

🤔 Hmmm… I'll restore the `await` because if artifact saving is no longer the last step, we might have unexpected "funny" things happening in the code